data/css/_mesh.scss

```diff
@@ -58,12 +58,9 @@ $display-font-composite: 'Roboto' !default;
 }
 
 .CardLegacyPolaroid {
-    &__icon {
-        padding-top: 10px;
-    }
-
     &__label {
         font-size: 0.9em;
+        padding: 0.50em 0em 0.50em 0.70em;
```

Comment:
I would suggest moving this entire section to `_legacyPolaroid.scss`, I can't think of a case where we would not want this padding.